piezo_web_app/PiezoWebApp/src/services/application_builder/template_populator.py

```diff
@@ -0,0 +1,70 @@
+class TemplatePopulator:
+
+    def build_template(self, validated_arguments):
+        if validated_arguments["language"] == "Python":
+            return self.populate_python_job_template(validated_arguments)
+        return self.populate_scala_job_template(validated_arguments)
```

Comment:
Assuming Scala by default makes me a little uneasy, even though we should have validated before reaching this point. Would it be possible to have a dictionary of language names to template methods instead?